packages/components/inputs/radio-input/src/radio-group.tsx

```diff
@@ -104,15 +112,23 @@ const Group = (props: TGroupProps) => {
   });
   if (props.direction === 'inline') {
     return (
-      <div aria-labelledby={props.id}>
+      <div
+        aria-labelledby={props.id}
+        aria-invalid={props['aria-invalid']}
+        aria-errormessage={props['aria-errormessage']}
```

Comment:
I think these aria attributes can only be used in `input` elements.

Reply:
From MDN 
```
The aria-invalid attribute can be used with any typical HTML form element, and is not limited to elements that have an ARIA role assigned.
```
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-invalid

If you still want me to remove it, please let me know

Reply:
Ah thanks, I missed that info. All good then 👌